include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a779f0.h

```diff
@@ -275,6 +275,21 @@
 #define FUNC_TCLK1                 IP0SR1(0, 1)
 #define FUNC_HSCK2                 IP0SR1(0, 2)
 #define FUNC_GP1_01                IP0SR1(4, 0)
+
+#define FUNC_MMC_SD_CLK            IP0SR1(0, FUNC_DUMMY)
```

Comment:
Well, this is not technically true. As you said in the commit message, those pin functions are not defined in IPSR registers, so it would be incorrect to say that `MMC_SD_CLK` is configured with bit 0 in `IP0SR1`. 

(Yes, I saw that MMC_SD_CLK is described under IP0SR1 section in the HW manual, but I tend to see this as a poor documentation choice).

Also, please take a look at note below, regarding `FUNC_DUMMY` value.

So, I want to propose introduce "dummy" register instead of "dummy function". This will fix both issues: potential use of function 0xF in the next SoC generation and incorrect assignment of all static functions to bit 0 of `IP0SR1`.